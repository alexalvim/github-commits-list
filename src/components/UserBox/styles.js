
import styled from 'styled-components';

import Colors from '../../colors';
import Spaces from '../../spaces';
import Typo from '../../typo'

const ContentWrapper = styled.div`
  align-items: center;
  background-color: ${Colors.mainColor};
  border-radius: 5px;
  color: ${Colors.lightestColor};
  display: flex;
  flex-direction: column;
  justify-content: center;
  padding: ${Spaces.double};
  width: 100%;

  @media (min-width: ${Spaces.tabletBreakpoint}) {
    width: 300px;
  }
`;

const Image = styled.img`
  border-radius: 50%;
  height: 180px;
  margin-bottom: ${Spaces.double};
  width: 180px;
`;

const Title = styled.h3`
  font-size: ${Typo.large};
`;

export {
  ContentWrapper,
  Image,
  Title
}