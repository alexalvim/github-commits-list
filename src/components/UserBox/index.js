import React from 'react';

import { ContentWrapper, Image, Title } from './styles';

export default ({ avatarUrl, login }) => (
  <ContentWrapper>
    <Image src={avatarUrl}/>
    <Title>{login}</Title>
  </ContentWrapper>
)