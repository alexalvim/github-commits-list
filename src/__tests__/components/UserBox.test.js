import React from 'react';
import { shallow } from 'enzyme';

import UserBox from '../../components/UserBox';

describe('Testing UserBox', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<UserBox avatarUrl='avatarTest' login='test' />);

    expect(wrapper).toMatchSnapshot();
  });
});