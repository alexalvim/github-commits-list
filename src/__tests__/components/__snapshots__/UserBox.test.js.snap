// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing UserBox should render correctly 1`] = `ShallowWrapper {}`;
